pkg/log/log.go

```diff
@@ -0,0 +1,60 @@
+package log
+
+import (
+	"github.com/g2a-com/klio/internal/log"
+)
+
+func setLevel(level string) {
```

Comment:
I would exclude setters, this should follow only the global config of logging, any customization in code should be discouraged